ui/admin/app/routes/scopes/scope/roles/role/scopes.js

```diff
@@ -0,0 +1,22 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { GRANT_SCOPE_KEYWORDS } from 'api/models/role';
+
+export default class ScopesScopeRolesRoleScopesRoute extends Route {
+  // =services
+
+  @service store;
+
+  async model() {
+    const role = this.modelFor('scopes.scope.roles.role');
+    console.log('role: ', role);
+    if (!GRANT_SCOPE_KEYWORDS.some((id) => role.grant_scope_ids.includes(id))) {
+      console.log('ENTERED');
```

Comment:
heads up line 12 and line 14 have leftover console logs

Reply:
Thanks. Removed